pyanaconda/payload/__init__.py

```diff
@@ -749,15 +524,14 @@ def _setupNFS(mountpoint, server, path, options):
             raise PayloadSetupError(str(e))
 
     ###
-    ### METHODS FOR INSTALLING THE PAYLOAD
+    # METHODS FOR INSTALLING THE PAYLOAD
     ###
     def preInstall(self):
```

Comment:
Would you consider to rename all payload methods with the camel case names, please?

Reply:
Heh, how can I missed that? Will fix that here.

Reply:
Fixed.